wee_slack.py

```diff
@@ -1911,7 +1911,7 @@ def change_suffix(self, new_suffix):
     def get_sender(self):
         name = ""
         name_plain = ""
-        if 'bot_id' in self.message_json and self.message_json['bot_id'] is not None:
+        if 'bot_id' in self.message_json and self.message_json['bot_id'] in self.team.bots:
```

Comment:
Maybe even better written as `if self.message_json.get('bot_id') in self.team.bots` ?